docs/preview.md

```diff
@@ -0,0 +1,48 @@
+# Preview
+
+Ruff includes an opt-in preview mode to help us collect community feedback and gain confidence that changes are a
```

Comment:
if i read "collect community feedback" it unfortunately sounds to me as if we're enabling telemetry in this mode

Reply:
I'll rephrase, thanks!